.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-18.04
     strategy:
       matrix:
-        python-version: [2.7, 3.3, 3.4, 3.5, 3.6, 3.7, 3.8, pypy2, pypy3]
+        python-version: [2.7, 3.3, 3.4, 3.5, 3.6, 3.7, 3.8, 3.9, 3.10-dev, pypy2, pypy3]
```

Comment:
I'd prefer to stick to just released Python versions.

Reply:
Understood.  It is a [release candidate](https://www.python.org/downloads/release/python-3100rc1/) with a fair number of changes so I wanted to give it a try but we can back off.